cocos/core/scene-graph/layers.ts

```diff
@@ -130,6 +131,34 @@ export class Layers {
       delete Layers.BitMask[Layers.BitMask[bitNum]];
       delete Layers.BitMask[bitNum];
   }
+
+  /**
+   * @en Given a layer name, returns the layer index as defined by either a Builtin or a User Layer in the Tags and Layers manager.
+   * @zh 给定层名称，返回由标记和层管理器中的内置层或用户层定义的层索引。
+   * @param layer Name's layer
```

Comment:
parameter name should be after `@param` decorator, 

`@param name layer's name`